test/test_jit.py

```diff
@@ -7645,12 +7645,9 @@ def forward(self, x, y):
     'test_var_dim_1d',
     'test_var_dim_1d_neg0',
     'test_var_dim_neg0',
-    'test_norm_inf',
-    'test_norm_inf_2_dim',
     'test_norm_fro',
     'test_norm_fro_default',
     'test_norm_nuc',
-    'test_renorm_norm_inf',
     'test_matrix_power_n=-1',  # involves inverse
     'test_matrix_power_n=-3',  # involves inverse
```

Comment:
I think the matrix power tests can be removed as well. These tests were added in earlier because they were implemented using `inverse`, which at that time was included in this list as well.